### ISSUE TITLE
Porting to python 3.5.

### ISSUE THREAD

**Issue by Pulkit07** [geekcomputers/Python#54]
I saw your files and they are not compatible with python 3. I will like to make them work with Python 3.5 too. Let me know if you want someone to work on it.

**Comment by rajeshkumargp** [comment]
Hi, Can i work on this ?

**Comment by geekcomputers** [comment]
That will be good as long as you don’t overwrite the 2.7 versions, can you do them separately?

Cheers
Craig

**Comment by geekcomputers** [comment]
That will be good as long as you don’t overwrite the 2.7 versions, can you do them separately 

Cheers
Craig

**Comment by Pulkit07** [comment]
Yes I can, I will be sending patches which will make the files run of versions>=2.6

**Comment by geekcomputers** [comment]
Excellent

**Comment by Mtrix-Blackspecter** [comment]
Can i pach you with a better code?

**Comment by geekcomputers** [comment]
What do you mean ?

**Comment by yuanchueh** [comment]
Can you also update the Readme file to indicate which versions of python the scripts were written and compatible for? 
When going through it would make things easier.

**Comment by geekcomputers** [comment]
Thank you for you contribution to the repo and the community

I wrote them all for Python 2.7

Cheers
Craig

**Comment by LittleLampLight** [comment]
Python 3 porting is all about print() and input() istead of raw_input()
